src/lib/kwio.sh

```diff
@@ -64,13 +64,13 @@ function colored_print()
     if [ -t 1 ]; then
       printf "$colored_format" "$message"
     else
-      printf '%s' "$message"
+      printf '%s' "$message" || true
     fi
   else
     if [ -t 1 ]; then
       printf "$colored_format\n" "$message"
     else
-      printf '%s\n' "$message"
+      printf '%s\n' "$message" || true
```

Comment:
I don't think you really need this. I tried to reproduce the error that you mentioned in the commit, but I did not see it. Maybe try to address my comment in the first commit, and remove this commit, it might work.